index.js

```diff
@@ -1943,6 +1943,29 @@
         return either.isRight ? [either.value] : [];
       }));
 
+  //# toEither :: a -> b? -> Either a b
+  //.
+  //. Takes a failure value, 'x', and a possibly null or undefined value, 'y'.
+  //. Returns a Right of 'y' if it is defined and not null, otherwise returns
+  //. a Left of 'x'.
```

Comment:
I find mention of "'x'" and "'y'" a bit confusing. I think we could get away with referring to "the first argument" and "the second argument".

How about this?

_Converts an arbitrary value to an Either: a Left if the value is `null` or `undefined`; a Right otherwise. The first argument specifies the value of the Left in the "failure" case._

I'd like to improve the descriptions of many of the library's functions. In many cases the description is just an English translation of the type signature. In this case, for example, a direct translation would be:

_Returns a Left of its first argument if its second argument is `null` or `undefined`; a Right of its second argument otherwise._

I like the mention of a "failure value" in your description. I'd like to avoid mention of "'x'" and "'y'" though.